components/QuestionCard.vue

```diff
@@ -158,5 +168,8 @@ export default class QuestionCard extends mixins(UserMixin) {
   image?: HTMLImageElement
 }
 </script>
-<style lang="scss">
```

Comment:
Wait don't set no padding on the body this is troll just do `<template slot="no-body></template>` and put all your elements in there to achieve no padding - this one will override other vxcard styles

Reply:
Refer to notification card